### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by fed1k**
* Added menu button ( Hamburger button )
* Added functionality to it 
* When user clicks button, navigation bar lists appear
* When user clicks x sign, they disappear
* When user chooses one of the options, user will be directed to that particular page

### CODE REVIEW

File: app.css [hunk 1]
```diff
@@ -14,66 +14,69 @@
...
 
-.nav-icon div {
-  width: 25px;
-  height: 4px;
-  background-color: #000;
-  margin: 4px;
-  transition: all 0.3s ease;
+.menu-list {
+  list-style-type: none;
 }
```
Comment (mateo951) [inline app.css]: There is an issue with the functionality of the hamburger button. When you close it, the desktop menu popups as you will see in the following image. Please be sure to fix this so you meet the project requirements 🙌

image

**Review by mateo951 — CHANGES_REQUESTED** [review]
### Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

File: app.css (hunk 1 above)
Comment (fed1k) [reply app.css]: Bro can you help to fix it? I have tried everything but nothing has worked

Comment (fed1k) [reply app.css]: @mateo951 Bro can you help to fix it? I have tried everything but nothing has worked

**Review by Whoistolu — APPROVED** [review]
### Hi @fed1k 

You have done a great job implementing the recommended changes.
Your project is complete!  There is nothing else to say other than... It's time to merge.

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.css (hunk 1 above)
Comment (mateo951) [reply app.css]: Just got your message 😥Good to know you found a solution!

**Pull request merged**